md/product-versioning.md

```diff
@@ -3,12 +3,24 @@ Discover how Bonita manages its product and artifacts versions and how it can im
 
 ## Product Versions
 
-Bonita follows the usual semantic versioning X.Y.z:
+After years of semantic versioning, Bonita now follows the calendar versioning: YYYY.#-mmdd, since version 2021.1:
+* YYYY is the year of the release
+* \# is the release number. Two versions are released every year.
```

Comment:
why an anti-slash?

Reply:
# is used for title \ is an escape character